Various technical notes/transition_infos.json

```
@@ -597,8 +586,8 @@
           "requires": []
         },
         {
-          "area_id": "0xDE524DA6",
-          "area_name": "Twin Outpost (Underwater Cave)",
+          "area_id": "0x239A2165",
+          "area_name": "Turtle Monument",
```

Good catch on the turtle monument, but twin outpost does still lead to underwater cave
```suggestion
          "area_id": "0x239A2165",
          "area_name": "Turtle Monument",
        },
        {
          "area_id": "0xDE524DA6",
          "area_name": "Twin Outpost (Underwater Cave)",
          "requires": []
```
If we want to just not deal with the transition weirdness thins brings for now, let's add a comment at the top of the file explaining why we don't handle it.

---

As mentioned in both the commit titles as well as the utils.py file that writes it to the Spoiler Log:
**Twin Outposts (Underwater Cave)** is temporarily removed from the randomization proccess. An essential part of that removal is that this exit is not included in the .json file.

---

I mark this as resolved because it's a duplicate of another conversation.